courses/machine_learning/deepdive2/image_classification/solutions/2_mnist_models.ipynb

```diff
@@ -19,7 +19,7 @@
   },
```

Comment:
<p>Maybe clear the output here since it has your bucket name in it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/705/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>